lib/basis_encode.cpp

```diff
@@ -607,6 +607,8 @@ ktxTexture2_CompressBasisEx(ktxTexture2* This, ktxBasisParams* params)
     ktx_uint32_t transfer = KHR_DFDVAL(BDB, TRANSFER);
     if (transfer == KHR_DF_TRANSFER_SRGB)
         cparams.m_perceptual = true;
+    else if (params->perceptual)
+        cparams.m_perceptual = params->perceptual; // We should probably warn in this case because basis doesn't like this perceptual with linear
```

Comment:
The BasisU m_perceptual param from the comment

```
// Use perceptual sRGB colorspace metrics instead of linear
```

is tied to sRGB input so I think this new `--perceptual` option should only apply to the astc encoder. Mind you a comment in the bc7 encoder that is included in the BasisU source says with `m_perceptual` error metrics are calculated in YCbCr instead of RGB. which suggests it is not entirely tied to sRGB. Let's ignore that.

Reply:
Remove these lines too.